### PR TITLE
mrc-2192 Make MINT homepage minimally accessible

### PR DESCRIPTION
**Pull request by mwoodbri**
* Add accessibility statement following guidance provided by Digital Accessibility Obligations Team.
  * Some parts of this are required boilerplate, some more specific to MINT. But written deliberately so that it can be a template for other apps.
  * Link only appears in navigation bar on the homepage.
* Fix issues on homepage

### CODE REVIEW

File: src/app/static/src/app/components/accessibilityPage.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                <p>We work to achieve and maintain <a href="https://www.w3.org/TR/WCAG21/">WCAG 2.1 AA standards</a>,
+                    but it is not always possible for all our
+                    content to be accessible. Where content is not accessible, we will state a reason, warn users and
+                    offer alternatives.</p>
+                <h2>Technical information about this website’s accessibility</h2>
+                <p>Imperial College London is committed to making its website accessible in accordance with the Public
+                    Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+                <p>This website is partially compliant with the <a href="https://www.w3.org/TR/WCAG21/">Web Content
+                    Accessibility Guidelines version 2.1</a> AA standard, due to the known issues listed above.</p>
+                <h2>Reporting accessibility issues</h2>
```
Comment (EmmaLRussell) [inline src/app/static/src/app/components/accessibilityPage.vue:33]: I guess this is the right thing, for an requests of this sort for MINT and others to go through Accessibility Support?

File: src/app/static/src/app/components/accessibilityPage.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                <p>If you need information on this website in a different format like accessible PDF, large print, easy
+                    read, audio recording or braille or if you find any accessibility issues not listed on this page
+                    then please contact accessibility@imperial.ac.uk.</p>
```
Comment (EmmaLRussell) [inline src/app/static/src/app/components/accessibilityPage.vue:36]: Could make this a mailto

File: src/app/static/src/app/components/accessibilityPage.vue [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                <p>We’ll consider your request and get back to you in 7 days.</p>
+                <h2>Enforcement procedure</h2>
+                <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies
+                    (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility
+                    regulations’). If you’re not happy with how we respond to your complaint, <a
+                        href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support
+                        Service (EASS)</a>.</p>
+                <h2>How we test this website</h2>
+                <p>This website was last tested for accessibility compliance on 22 February 2021, and these tests have
+                    been carried out internally using the <a href=https://accessibilityinsights.io/en/">Accessibility
```
Comment (EmmaLRussell) [inline src/app/static/src/app/components/accessibilityPage.vue:46]: Missing an opening " on href

**Review by EmmaLRussell — COMMENTED** [review]
Looks good! 

Where did you source the template for the accessibility statement. I don't think we ever got the original attachment to the email from Accessibility Support, and this looks a iittle bit different to the Imperial example [here](https://www.imperial.ac.uk/about-the-site/accessibility/) (though very similar!)

I still get some colour contrast errors on accessibility insights when run against the region page showing interventions, along with a few little things like duplicate ids and inaccessible select names, which aren't mentioned in the accessibility page but seem like they would be pretty easy to fix. Want to do some or all of those in separate tickets?

Screenshot from 2021-02-23 10-44-18
Screenshot from 2021-02-23 10-43-50

File: src/app/static/src/app/components/accessibilityPage.vue (hunk 1 above)
Comment (mwoodbri) [reply src/app/static/src/app/components/accessibilityPage.vue:33]: Spoke to Rich - he's happy for these to come direct so I've amended.

File: src/app/static/src/app/components/accessibilityPage.vue (hunk 2 above)
Comment (mwoodbri) [reply src/app/static/src/app/components/accessibilityPage.vue]: Thanks. I slavishly copied non-hyperlinked version from template. But now a link!

File: src/app/static/src/app/components/accessibilityPage.vue (hunk 3 above)
Comment (mwoodbri) [reply src/app/static/src/app/components/accessibilityPage.vue]: D'oh. Always pass final version through HTML validator!

**Comment by mwoodbri** [comment]
> Where did you source the template for the accessibility statement. I don't think we ever got the original attachment to the email from Accessibility Support, and this looks a iittle bit different to the Imperial example [here](https://www.imperial.ac.uk/about-the-site/accessibility/) (though very similar!)

I tried to combine the most relevant/helpful parts of that example (which isn't actually very well written) and the canonical guidance available [here](https://www.imperial.ac.uk/equality/activities/digital-accessibility/accessibility-statements/).

> I still get some colour contrast errors on accessibility insights when run against the region page showing interventions, along with a few little things like duplicate ids and inaccessible select names, which aren't mentioned in the accessibility page but seem like they would be pretty easy to fix. Want to do some or all of those in separate tickets?

Very happy to do this, but I think it should be done in a second pass, or even only in response to specific requests i.e. where was can get more concrete guidance. Some of it would require slightly more involved (i.e. not purely cosmetic) fixes, and even if resolved I'm not sure that outputs themselves (i.e. the charts) are fundamentally very accessible. So unsure this is easy to justify amongst other immediate priorities but I have created a maintenance ticket (mrc-2208)... and we should be more careful about considering this up-front for new apps.

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**